packages/core/src/ceramic.ts

```diff
@@ -287,6 +290,8 @@ class Ceramic implements CeramicApi {
     const logger = LoggerProvider.makeDiagnosticLogger(loggerConfig)
     const pubsubLogger = LoggerProvider.makeServiceLogger("pubsub", loggerConfig)
 
+    logger.imp(`Starting Ceramic node at version ${packageJson.version} with config: \n${JSON.stringify(config, null, 2)}`)
```

Comment:
Wouldn't this output the users private key which is usually contained in the `~/.ceramic/config.json`?

Reply:
Can you point me to where this private key information is loaded?  I'm not seeing it obviously within the [CeramicConfig](https://github.com/ceramicnetwork/js-ceramic/blob/be2b472640d35165efeb61437b83a81f29e878be/packages/core/src/ceramic.ts#L50).  Maybe it's within the `didProvider` object?  If we know which fields in the config may contain sensitive information, we could add a `redactSensitiveConfigInfo()` method that we call before logging the config to filter such fields out